### PR TITLE
skv with space

### PR DESCRIPTION
**Pull request by double-y**
### Description
Fix https://github.com/exploratory-io/tam/issues/4126

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [x] Pass devtools::check()
- [x] Pass devtools::test()
- [x] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/pairwise.R [hunk 1]
```diff
@@ -50,7 +50,11 @@ do_cosine_sim.kv <- function(df, subject, key, value = NULL, distinct=FALSE, dia
     df
   }
 
-  (df %>% dplyr::do_(.dots=setNames(list(~calc_doc_sim_each(.)), cnames[[1]])) %>%  tidyr::unnest_(cnames[[1]]))
+  # Use cnames[[3]] for temp column name to avoid name conflict with grouped colum name
```
Comment (hideaki) [inline R/pairwise.R]: typo: colum

Comment (hideaki) [inline R/pairwise.R]: Please explain some more on what is this cnames[[3]] used for and why.

Comment (double-y) [inline R/pairwise.R]: This is actually confusing, so I'll use "tmp" column for temporary column

File: R/pairwise.R [hunk 2]
```diff
@@ -50,7 +50,10 @@ do_cosine_sim.kv <- function(df, subject, key, value = NULL, distinct=FALSE, dia
     df
   }
 
-  (df %>% dplyr::do_(.dots=setNames(list(~calc_doc_sim_each(.)), cnames[[1]])) %>%  tidyr::unnest_(cnames[[1]]))
+  tmp_col <- avoid_conflict(grouped_column, "tmp")
```
Comment (hideaki) [inline R/pairwise.R]: It's good that this is done in simpler way, but still these line deserves a comment on what they are doing.

**Pull request merged**